contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -342,7 +342,9 @@ Do not use fields that you don't need - simpler is better.
 Use of the `Reason` field is encouraged.
 
 Use the `LastHeartbeatTime` with great caution - frequent changes to this field
-can cause a large fan-out effect for some resources.
+can cause a large fan-out effect for some resources. The name of this field also
+varies from resource to resource, sometimes it's called `LastProbeTime`, and other
+times it's called `LastUpdateTime`.
```

Comment:
Might be worth providing examples of where `LastProbeTime` (PodCondition, PersistentVolumeClaimCondition) and `LastUpdateTime` (DeploymentCondition) are used.

Reply:
Should I just mention them or link to them in the documentation?

Reply:
I think just mentioning them is fine.